source/MRMesh/MRId.h

```diff
@@ -147,10 +147,10 @@ inline constexpr Id<T> operator - ( Id<T> id, unsigned int a ) { return Id<T>{ i
 template <typename T>
 inline constexpr Id<T> operator - ( Id<T> id, size_t a )       { return Id<T>{ id.get() - a }; }
 
-inline constexpr FaceId operator "" _f( unsigned long long i ) noexcept { return FaceId{ (int)i }; }
-inline constexpr VertId operator "" _v( unsigned long long i ) noexcept { return VertId{ (int)i }; }
-inline constexpr EdgeId operator "" _e( unsigned long long i ) noexcept { return EdgeId{ (int)i }; }
-inline constexpr UndirectedEdgeId operator "" _ue( unsigned long long i ) noexcept { return UndirectedEdgeId{ (int)i }; }
-inline constexpr VoxelId operator "" _vox( unsigned long long i ) noexcept { return VoxelId{ size_t( i ) }; }
+inline constexpr FaceId operator ""_f( unsigned long long i ) noexcept { return FaceId{ (int)i }; }
+inline constexpr VertId operator ""_v( unsigned long long i ) noexcept { return VertId{ (int)i }; }
+inline constexpr EdgeId operator ""_e( unsigned long long i ) noexcept { return EdgeId{ (int)i }; }
+inline constexpr UndirectedEdgeId operator ""_ue( unsigned long long i ) noexcept { return UndirectedEdgeId{ (int)i }; }
+inline constexpr VoxelId operator ""_vox( unsigned long long i ) noexcept { return VoxelId{ size_t( i ) }; }
```

Comment:
why space removed?

Reply:
it leads to `"-Wdeprecated-literal-operator"` in new clang